tasks/util/sort_object.js

```diff
@@ -0,0 +1,21 @@
+function caseInsensitive(a, b) {
+    return a.toLowerCase().localeCompare(b.toLowerCase());
+}
+
+function sortObject(obj) {
+    var allKeys = Object.keys(obj);
+    allKeys.sort(caseInsensitive);
+
+    var newObj = {};
+    for(var i = 0; i < allKeys.length; i++) {
+        var key = allKeys[i];
+        var v = obj[key];
+        newObj[key] = (typeof v === 'object' && v !== null && !(v instanceof Array)) ?
```

Comment:
Actually we need to dig into arrays too, right? We've got stuff in the schema like:

```
      "items": [
       {
        "valType": "number",
        "min": 0,
        "max": 1,
        "editType": "calc"
       },
       {
        "valType": "number",
        "min": 0,
        "max": 1,
        "editType": "calc"
       }
      ],
```

Reply:
Good call.
https://github.com/plotly/plotly.js/pull/5776/commits/aea30228e2fa0a15f8dcbccf728b2553d035470f also fixed cases where arrays e.g. *enumerated* `valType` turned into objects!
Before:
```json
   "values": {
    "0": "forward",
    "1": "reverse"
   }
```
After
```json   
   "values": [
    "forward",
    "reverse"
   ]
```